build/maven.settings.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings
+	xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd"
+	xmlns="http://maven.apache.org/SETTINGS/1.1.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<servers>
+		<server>
+			<username>${env.ARTIFACTORY_USER}</username>
+			<password>${env.ARTIFACTORY_KEY}</password>
+			<id>central</id>
+		</server>
+		<server>
+			<username>${env.ARTIFACTORY_USER}</username>
+			<password>${env.ARTIFACTORY_KEY}</password>
+			<id>snapshots</id>
+		</server>
+		<server>
+			<username>${env.ARTIFACTORY_USER}</username>
+			<password>${env.ARTIFACTORY_KEY}</password>
+			<id>releases</id>
+		</server>
+    </servers>
+	<profiles>
+		<profile>
+			<id>Merative_Artifactory</id>
+			<repositories>
+                <repository>
+					<id>central</id>
+					<url>https://artifactory.commops.truvenhealth.com:443/artifactory/wh-de-id-maven-virtual</url>
```

Comment:
Do we still have a path to build that works for the public in general ? This one has the internal Artifactory hardcoded in

Reply:
I will change the action that runs on pull-requests to not use the internal Artifactory and the action that runs on push does not run on forks.  That should allow external users to get the benefits of the checks provided by the maven build without having to make build-related changes.